runtime/lib/capi/RuntimeCAPI.cpp

```diff
@@ -130,24 +131,24 @@ void pyregistry(int64_t identifier)
     //
     // This function cannot be tested from the runtime tests because there would be no valid python
     // function to callback...
-
-    std::string libpath = LIBREGISTRY;
-
-    void *handle = dlopen(libpath.c_str(), RTLD_LAZY);
+    void *handle = dlopen(LIBREGISTRY, RTLD_LAZY);
```

Comment:
Could explain where is LIBREGISTRY compile time macro defined?

Reply:
I'll add a comment. It is added during the creation of the runtime.

Reply:
https://github.com/PennyLaneAI/catalyst/pull/596/commits/bed566f034cef9305cde4d2ed3bd2b5f8baa7d0a